parser/encoding/rpc.go

```diff
@@ -69,43 +91,47 @@ func (e *RPCEncoding) RequestEncodingForMethod(method string) *RequestEncoding {
 	return nil
 }
 
-// RequestEncoding expresses how a request should be encoded for a specified set of HTTPMethods
+// ResponseEncoding expresses how a response should be encoded on the wire
+type ResponseEncoding struct {
+	// Contains metadata about how to marshal a HTTP parameter
+	Fields []*ParameterEncoding
+}
+
+// RequestEncoding expresses how a request should be encoded for an explicit set of HTTPMethods
 type RequestEncoding struct {
 	// The HTTP methods these field configurations can be used for
 	HTTPMethods []string
 	// Contains metadata about how to marshal a HTTP parameter
 	Fields []*ParameterEncoding
 }
 
-// ParameterEncoding expresses how a parameter should be encoded
+// ParameterEncoding expresses how a parameter should be encoded on the wire
 type ParameterEncoding struct {
 	// Location (e.g. header, query, body) where the parameter should be marshalled from/to
 	Location ParameterLocation
 	// The location specific name of the parameter (e.g. cheeseEater, cheese-eater, X-Cheese-Eater
 	Name string
+	// Whether the name should be case-sensitive
+	CaseSensitive bool
```

Comment:
When sent to `true` the standard library JSON (i.e. the one we use in the client generator) will still be case insensitive. Is this ok? (i.e. what's the use case fo having this be true?)

Reply:
Good point. I'll remove it